.eslintrc.js

```diff
@@ -42,6 +42,20 @@ module.exports = {
 				],
 			},
 		],
+		'no-unused-vars': [
+			'error',
+			{
+				varsIgnorePattern: 'React',
+			},
+		],
+		'@typescript-eslint/no-unused-vars': [
+			'error',
+			{
+				varsIgnorePattern: 'React',
+			},
+		],
+		'react/react-in-jsx-scope': 'off',
+		'testing-library/no-unnecessary-act': 'off',
```

Comment:
> To avoid file conflicts with coming PRs (due to new rules alerts), some rules are being disabled temporarily. 

How about adding a comment over the temporary rules mentioning these additions are temporary (so that we remember to remove them later)

Reply:
Oh yeah, it makes sense. I just added it along with a TODO to make it easier to see when we open the file in an editor.